equinox/_tree.py

```diff
@@ -51,6 +51,22 @@ def count(self, item):
         return self._count[id(item)]
 
 
+class _DistinctTuple:
+    def __init__(self, contents=()):
+        self.contents = tuple(contents)
+
+    def __eq__(self, other):
+        if isinstance(other, _DistinctTuple):
+            return self.contents == other.contents
+        return False
+
+    def __hash__(self):
+        return hash(self.contents)
+
+    def __repr__(self):
+        return "Distinct Tuple: " + repr(self.contents)
```

Comment:
I think a lot of this is unused -- just
```python
class _DistinctTuple:
    pass
```
should suffice, I think?

(In particular I don't think reprs are ever user-visible, and the `contents` are always empty.)

Reply:
That's true, it was a hold over from my print debugging